tests/core/test_ai_circuit_breaker.py

```
@@ -0,0 +1,308 @@
+import pytest
+import asyncio
+import random
+from datetime import datetime, timedelta
+from app.core.ai_circuit_breaker import (
+    AICircuitBreaker,
+    CircuitState,
+    FailureEvent,
+    AICircuitStats,
+    registry,
+    ai_circuit_protected,
+    CircuitBreakerError
```

Ruff (0.8.2)

Trailing comma missing

(COM812)